capitalize_decorator.rb

```diff
@@ -0,0 +1,7 @@
+require_relative 'decorator'
+
+class CapDecorator < Decorator
+  def correct_name
+    @nameable.correct_name.upcase
```

Comment:
- [ ] You are making the name uppercase (all letters are in capital), instead of capitalized (only the first letter is capital). Kindly fix that.

  ```
  @nameable.correct_name.capitalize
  ```